tests/test_module_mock.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from loguru import logger
+import mmc
+import PIL
+import torch
+
+
+def test_init():
+    from mmc.mock.openai import MockOpenaiClipModule
+    from mmc.loaders import OpenAiClipLoader
+
+    ldr = OpenAiClipLoader(id='RN50')
+    clip = MockOpenaiClipModule(ldr)
+
+def test_available_models():
+    from mmc.mock.openai import MockOpenaiClipModule
+    from mmc.loaders import OpenAiClipLoader
+
+    ldr = OpenAiClipLoader(id='RN50')
+    clip = MockOpenaiClipModule(ldr)
+    assert clip.available_models() == str(ldr)
+
+def test_private_mmc():
+    from mmc.mock.openai import MockOpenaiClipModule, MockOpenaiClip
+    from mmc.loaders import OpenAiClipLoader
+    from mmc.multimodalcomparator import MultiModalComparator
+
+    ldr = OpenAiClipLoader(id='RN50')
```

Comment:
I feel having to choose which loader to use defeats the purpose of automation that mocking the CLIP module could allow for. What if instead of 
`ldr = OpenAiClipLoader(id='RN50')`
We had something like 
`ldr = OpenMMCLoader(id='[clip - sajjjadayobi - clipfa]')`
That accepts and is compatible to all valid MMC perceptors

The main idea of `mock_api_plus` was that it would carry the spirit of  `mock_api`, where when we mock OpenAI we still allow for _any_ of the supported perceptors to be loaded to it with just  - so I just wanted to extend that so the user can be able to use the tokenizer and image preprocessor of all models without having to create a lot of conditions on their end